src/solvers/smt2/smt2_solver.cpp

```diff
@@ -162,6 +178,13 @@ void smt2_solvert::command(const std::string &c)
         std::cout << e.pretty() << '\n'; // need to do an 'smt2_format'
```

Comment:
Hmm, just noticed this one: we already have `smt2_format` by now.

Reply:
Well, yes, as of #3461. Will tweak.